examples/glusterfs/README.md

```diff
@@ -0,0 +1,44 @@
+## Glusterfs
+
+[Glusterfs](http://www.gluster.org) is an open source scale-out filesystem. These examples provide information about how to allow Docker containers use Glusterfs volumes.
+
+The examples consist of:
+- A pod that runs on hosts that install Glusterfs client package.
+- A pod that runs on hosts that don't install Glusterfs client package and use [Super Privileged Container](http://developerblog.redhat.com/2014/11/06/introducing-a-super-privileged-container-concept/) to mount Glusterfs.
```

Comment:
Suggest leaving out the SPC example from the initial PR -- require preinstalled GlusterFS client.  
Later, we can iterate on SPC and Security Contexts and such.